src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_ARM/ARMEmitter.cs

```diff
@@ -18,31 +18,55 @@ public ARMEmitter(NodeFactory factory, bool relocsOnly)
         public ObjectDataBuilder Builder;
         public TargetRegisterMap TargetRegister;
 
+        // check the value length
+        private bool isBitNumOverflow(int value, byte numBits)
```

Comment:
Could you please use CamelCase to name these?

Also, these methods can be static.